prow/plugins/golint/golintsuggestion/golint_suggestion.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package golintsuggestion
+
+import (
+	"regexp"
+	"fmt"
+	"strconv"
+	"strings"
+	"github.com/golang/lint"
+)
+
+const (
+    lintNamesRegex1 = "don't use underscores in Go names; (.*) should be (.*)"
+    nameUnderscore = "NAME_UNDERSCORE"
+)
+
+var lintCaseMap = map[string]string {
+	lintNamesRegex1: nameUnderscore,
+}
+
+func CanSuggestCodeChange(p lint.Problem) bool {
+	fmt.Printf("\n\n***problem:" + p.Text) //don't use underscores in Go names; func Qux_1 should be Qux1
+	for r, _ := range lintCaseMap {
+		result, _ := regexp.MatchString(r, p.Text)
+		if result {
+			return true
+		}
+	}
+	return false
+   }
+
+func SuggestCodeChange(p lint.Problem) string {
```

Comment:
please add a [godoc](
https://blog.golang.org/godoc-documenting-go-code) comment to this method